rules/linux/execution_egress_connection_from_entrypoint_in_container.toml

```diff
@@ -0,0 +1,76 @@
+[metadata]
+creation_date = "2024/07/10"
+integration = ["endpoint"]
+maturity = "production"
+updated_date = "2024/07/10"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies a sequence of events where a process named `entrypoint.sh` is started in a container, followed by a
+network connection attempt. This sequence indicates a potential egress connection from an entrypoint in a container. An
+entrypoint is a command or script specified in the Dockerfile and executed when the container starts. Attackers can use
+this technique to establish a foothold in the environment, escape from a container to the host, or establish persistence.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Egress Connection from Entrypoint in Container"
+risk_score = 47
+rule_id = "c75d0c86-38d6-4821-98a1-465cff8ff4c8"
+severity = "medium"
+tags = [
+    "Domain: Endpoint",
```

Comment:
Should we add `Domain: Container` here as well? May help associate endpoint rules with docker unless free searching for "docker" itself.